src/AssimpWorker/model/Folder.cpp

```diff
@@ -71,19 +71,29 @@ namespace Model {
 		return childFolders;
 	}
 
+	void Folder::removeAttribute(std::string key){
```

Comment:
Am I seeing this right that this is only there so Attributes can be used as temporary storage for the import process? Might it be sensible to have a more generic facility for this? I would like to avoid the attribute namespace being polluted with temps.